packages/core/useGeolocation/index.ts

```diff
@@ -4,11 +4,15 @@ import { ref, Ref } from 'vue-demi'
 import { tryOnMounted, tryOnUnmounted } from '@vueuse/shared'
 import { ConfigurableNavigator, defaultNavigator } from '../_configurable'
 
-export function useGeolocation(positionOptions: PositionOptions = {
+interface PositionOpts extends PositionOptions, ConfigurableNavigator {}
+
+export function useGeolocation(positionOptions: PositionOpts = {
   enableHighAccuracy: true,
   maximumAge: 30000,
   timeout: 27000,
-}, { navigator = defaultNavigator }: ConfigurableNavigator = {}) {
+  navigator: defaultNavigator,
+}) {
```

Comment:
The default value was not correct, partial options will override the entire default value. Should change to destructuring default assignments. This was a mistake, it would be nice if you can fix it alongside. Thanks